src/app/components/Companies.js

```diff
@@ -29,7 +29,10 @@ class Companies extends React.Component {
         <Link
           to={`companies/${company.id}`}
           key={company.id}
-          style={{ display: showOnlyCertified && company.score < 6 ? 'none' : 'block' }}
+          style={{
+            display:
+              showOnlyCertified && !(company.score / checklist.length > 0.6) ? 'none' : 'block'
```

Comment:
good catch! not a big deal, but i think you can also do: 
```suggestion
              showOnlyCertified && (company.score / checklist.length < 0.6) ? 'none' : 'block'
```